app/models/blood_sugar.rb

```diff
@@ -31,19 +31,40 @@ class BloodSugar < ApplicationRecord
 
   scope :for_v3, -> { where(blood_sugar_type: V3_TYPES) }
   scope :for_sync, -> { with_discarded }
+  scope :for_recent_measures_log, -> do
+    recorded_date = "DATE(recorded_at at time zone 'UTC' at time zone '#{CountryConfig.current[:time_zone]}')"
+    order(Arel.sql("#{recorded_date} DESC, recorded_at ASC"))
+  end
 
   THRESHOLDS = {
-    high: {random: 300,
-           post_prandial: 300,
-           fasting: 200,
-           hba1c: 9.0}
+    bs_below_200:
+      {random: 0..200, post_prandial: 0..200, fasting: 0..126, hba1c: 0..7},
+    bs_200_to_300:
+      {random: 200..300, post_prandial: 200..300, fasting: 126..200, hba1c: 7..9},
+    bs_over_300: {
+      random: 300..Float::INFINITY,
+      post_prandial: 300..Float::INFINITY,
+      fasting: 200..Float::INFINITY,
+      hba1c: 9..Float::INFINITY
+    }
```

Comment:
If I understood correctly
- This adds the notions of `bs_below_200` and `bs_200_to_300`.
- It does away with the `high` terminology and calls it `bs_over_300` now.
- We're adding it to support showing the states of individual blood sugars in the activity log.

Is that correct?

Reply:
Yes.